middleware/dnssec/README.md

```diff
@@ -43,5 +43,7 @@ If monitoring is enabled (via the *prometheus* directive) then the following met
 
 * coredns_dnssec_cache_size{type} - total elements in the cache, type is "signature".
```

Comment:
No, not nit-picking at all.  This is something I noticed and thought about fixing. :-)